tmk_core/arm_atsam.mk

```diff
@@ -54,3 +54,7 @@ EXTRALIBDIRS =
 bin: $(BUILD_DIR)/$(TARGET).hex
 	$(OBJCOPY) -Iihex -Obinary $(BUILD_DIR)/$(TARGET).hex $(BUILD_DIR)/$(TARGET).bin
 	$(COPY) $(BUILD_DIR)/$(TARGET).bin $(TARGET).bin;
+	@if $(AUTOGEN); then \
+		$(REMOVE) $(KEYMAP_PATH)/$(TARGET).hex | true; \
```

Comment:
```suggestion
		$(REMOVE) $(KEYMAP_PATH)/$(TARGET).$(FIRMWARE_FORMAT) | true; \
```

Reply:
This needs to remove the hex file, so it's not copied to `qmk.fm/compiled`.  
Otherwise, `travis_compiled_push.sh` will push a hex file where we need bin files for those boards.

Reply:
Specifically, I tried this, in the core "hex" target, and it caused this issue:
https://travis-ci.org/qmk/qmk_firmware/builds/490409457#L1203-L1208

And:
https://travis-ci.org/qmk/qmk_firmware/builds/490409457#L1303-L1305